CONTRIBUTING.md

```diff
@@ -9,9 +9,12 @@
 * [Idle Pull Requests Policy](#idle)
 * [Unresolved Pull Requests Policy](#unresolved)
 
+This document is an extension to the generic [Anuket Project Operations and Procedures](https://wiki.anuket.io/display/HOME/Anuket+Project+Operations+and+Procedures) and describes contribution rules to the Anuket specifications, that are the set of documents maintained in this repository. 
+
 <a name="permissions"></a>
 ### Permission levels
```

Comment:
Workstream to be changed by sub-project in this section

Reply:
Workstreams corrected in [fc58bc5](https://github.com/cntt-n/CNTT/pull/2805/commits/fc58bc510b0c6520b48196696396b8c32f94d641)